go.mod

```
@@ -13,6 +13,7 @@ require (
 	github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
 	github.com/fasthttp/router v1.0.4
 	github.com/fasthttp/session v1.1.8
+	github.com/fasthttp/session/v2 v2.0.2
```

Don't think we need this anymore:

```suggestion
	github.com/fasthttp/session/v2 v2.0.2
```